### PR TITLE
Update docs and transition from JSON to YAML

### PR DESCRIPTION
**Pull request by lmh91**
Docs PR build: https://juliaphysics.github.io/SolidStateDetectors.jl/previews/PR177/

### CODE REVIEW

File: docs/src/man/primitives.jl [hunk 1]
```diff
@@ -4,7 +4,8 @@ using SolidStateDetectors
 import SolidStateDetectors.ConstructiveSolidGeometry as CSG
 using SolidStateDetectors.ConstructiveSolidGeometry: ClosedPrimitive
 using SolidStateDetectors.ConstructiveSolidGeometry: Box
-using Plots
+using Plots; gr()
+Plots.reset_defaults()
```
Comment (fhagemann) [inline docs/src/man/primitives.jl:8]: What does this do?

Comment (lmh91) [reply docs/src/man/primitives.jl:8]: No it didn't...

Comment (lmh91) [reply docs/src/man/primitives.jl:8]: 30e6216 solved it. I removed the `Plots.reset_defaults` again (force push).

**Comment by lmh91** [comment]
Okay, it is ready I guess. 
But it reminded me that I wanted to put the implementation of
proper logging, debug and warning outputs onto the ToDo list...

File: docs/src/man/primitives.jl (hunk 1 above)
Comment (fhagemann) [reply docs/src/man/primitives.jl:8]: well, `Plots.reset_defaults()` still seems to be in that file..

Comment (lmh91) [reply docs/src/man/primitives.jl:8]: Ah, right. It was in `primitives.jl` as well... We can clean this up later.

**Pull request merged**